src/lib/openapi/spec/group-schema.ts

```diff
@@ -7,49 +7,67 @@ export const groupSchema = {
     type: 'object',
     additionalProperties: true,
     required: ['name'],
+    description: 'A detailed information about a user group',
     properties: {
         id: {
-            type: 'number',
+            description: 'The group id',
+            type: 'integer',
+            example: 1,
```

Comment:
Not blocking and not really important on response schemas, but what's the minimum ID we use? We could indicate that here 💁🏼 

```suggestion
            example: 1,
            minimum: 1, // or 0?
```

Reply:
In my DB it's 1 but it's a DB implementation detail so not sure if it adds much value

Reply:
Yeah, and because it's not something you can set yourself, it doesn't matter much. I'd set it to `0` so that people can used unsigned integers if they want to, but I'll leave it up to you.

Reply:
Actually 0 should never occur because postgres starts counting from 1 at least on my machine. But I don't want to make too many assumptions here